webapp/channels/src/components/suggestion/emoticon_provider.test.tsx

```diff
@@ -83,18 +94,27 @@ describe('components/EmoticonProvider', () => {
         const pretext = ':blocklisted';
         const recentEmojis = ['blocklisted-1'];
 
-        const blocklistedEmojis = EMOJI_CATEGORY_SUGGESTION_BLOCKLIST.map((category, index) => {
-            const name = `blocklisted-${index}`;
-            return [name, {name, category}];
-        });
-        const customEmojisWithBlocklist = new Map([
+        const blocklistedEmojis: Array<[string, any]> = EMOJI_CATEGORY_SUGGESTION_BLOCKLIST.map(
```

Comment:
Why any? And... is it the type definition needed here?

Reply:
PS: Typescript was complaining if I didn't give a type definition.